python/dolfinx/io/gmshio.py

```diff
@@ -40,8 +43,10 @@ def ufl_mesh(gmsh_cell: int, gdim: int) -> ufl.Mesh:
     """
     shape, degree = _gmsh_to_cells[gmsh_cell]
     cell = ufl.Cell(shape, geometric_dimension=gdim)
-    scalar_element = ufl.FiniteElement("Lagrange", cell, degree, variant="equispaced")
-    return ufl.Mesh(ufl.VectorElement(scalar_element))
+
+    element = basix.ufl_wrapper.create_vector_element(
+        "Lagrange", cell.cellname(), degree, basix.LagrangeVariant.equispaced, dim=gdim, gdim=gdim)
```

Comment:
Is there a Basix `enum` for Lagrange?

Reply:
Yes, although the `create_*_element` convenience functions can also accept strings which are then converted using functions like `basix.finite_element.string_to_family`.

In this case, I've changed `"Lagrange"` to `basix.ElementFamily.P`

`cell.cellname()` is also a string, but I think it's best to leave it as so as the alternative is something ugly like `attr(basix.CellType, cell.cellname())`